### DISCUSSION TITLE
No login user

### DISCUSSION THREAD

**Discussion by PaulRohlik** [frangoteam/FUXA#899] [Ideas]
1)
Hello,
Would it be possible to add a non logged in user option?
Password for editor only.

2)
If I set a password on the button and log out, the button disappears. Why?
Is it possible to set the button to remain visible and ask for a password when clicked by an unauthorized person?

**Comment by unocelli** [comment]
Hi, you have to manage the users, only the Administrator have access to editor.
In header bar you have the button.
![image](https://github.com/frangoteam/FUXA/assets/48684550/ba9e5bff-bebb-43f6-958e-7eb5a0e13a86)

**Reply by PaulRohlik** [reply]
Yes, only admin can access the editor, that's fine. But can permanently logged in users be added as default? for example "viewer"? Or extend the expiration time see image to "never"?

The problem is that when no user is logged in, the HMI animation stops and nothing can be controlled.
![expirace](https://github.com/frangoteam/FUXA/assets/48689104/4580ac61-4bed-4953-a7ca-46068c9c9f11)

**Reply by unocelli** [reply]
You can Disable it, and than next time that you logged you receive a token without expiration

**Reply by PaulRohlik** [reply]
hi
yes, but if I disable it, the editor is enabled without the need to enter a password.

**Reply by unocelli** [reply]
Hi, sorry, you are right I still have to add the option that you mean

**Reply by PaulRohlik** [reply]
Thank you very much :)

**Reply by unocelli** [reply]
Hi, I added the option

**Comment by feribudi2** [comment]
I installed Fuxa on Docker, how do I set it to add a login username and password?

**Comment by feribudi2** [comment]
i didn't find settings.js' file

**Comment by PaulRohlik** [comment]
Hi,

usr and pass in: editor>setup>users
and enabled in: 

![image](https://github.com/frangoteam/FUXA/assets/48689104/04c919a2-e545-4c09-88ea-211362a2d165)
